tests/testthat/test-sparsegl.R

```diff
@@ -1,8 +1,285 @@
+library(Matrix)
+library(glmnet)
+library(gglasso)
+library(SGL)
+
 test_that("multiplication works", {
+  expect_equal(2 * 2, 4)
+})
 
+test_that("x is not a matrix", {
```

Comment:
1. The name needs to be more descriptive. You're testing that "sparsegl fails when x is not a matrix".
2. You can bundle a bunch of similar tests together. So this one down to about line 83, are all "input parameter mismatches cause failures" or something like that.
3. There's no need to test the objects you create. You made `x` a vector, so we don't test that it's not a matrix, just that `sparsegl` fails.